src/pages/AddPhoto/AddPhoto.tsx

```diff
@@ -1,8 +1,26 @@
 import React from 'react';
 import './AddPhoto.css';
+import { NavButton } from 'components/NavButton/NavButton';
+import { UploadPhoto } from 'components/UploadPhoto/UploadPhoto';
 
-export const AddPhoto: React.FC = () => {
+const button: Button = { text: "START TRACING", to: "/trace/instructions" };
+
+interface AddPhotoProps {
```

Comment:
Should we keep interfaces in the types.d.ts file?

Reply:
I like having the interfaces on the same page, because having the type of the props right there when we code is nice.

Reply:
Ok